Various/amagalma_ReaNoir track-item-take coloring utility.lua

```diff
@@ -27,6 +27,8 @@
 
 --[[
  * Changelog:
+ * v.161 (2017-03-26)
```

Comment:
Changelog line says `v.161`: it won't match with `1.61`. If you don't already know, note that there's another changelog syntax which doesn't require keeping the old ones: https://github.com/cfillion/reapack-index/wiki/Packaging-Documentation#changelog.